docs/All/classes.md

```diff
@@ -1 +1,16 @@
+---
+title: Recommended Classes
+tags:
+  - CAS
+  - Tandon
+  - Classes
+  - Planning
+---
+
 # Recommended Classes
+
+A list of recommended classes by former students. If you would like to add/update this list, send a [pull request](https://github.com/BUGS-NYU/nyu-cs-wiki/pulls)!
+
+## CAS
+
+## Tandon
```

Comment:
CAS Examples:
- Open Source Software Development

Tandon Examples:

- Distributed Systems 
- Algorithmic Machine Learning and Data Science (offered in fall)
- Programming Languages

Reply:
Do you have some more information on the courses? I think it would be good to have a short description and why its recommended.